src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -126,6 +139,31 @@
     }
 
     /**
+     * This method determines if a check should process javadoc containing non-tight html tags.
+     * This method must be overridden in checks extending {@code AbstractJavadocCheck} which
+     * are not supposed to process javadoc containing non-tight html tags.
+     *
+     * @return true if the check should or can process javadoc containing non-tight html tags;
+     *     false otherwise
+     * @see ParseStatus#isNonTight()
+     * @see <a href="http://checkstyle.sourceforge.net/writingjavadocchecks.html#Tight-HTML_rules">
+     *     Tight HTML rules</a>
+     */
+    public boolean doesCheckAcceptJavadocWithNonTightHtml() {
+        return true;
+    }
```

Comment:
@Vladlis and @rnveach ,
the more I look at this method name the more I dislike it.
Is it better to simply use `acceptJavadocWithNonTightHtml` ?

Reply:
@romani I agree with you. Renamed.